webapi/src/main/scala/org/knora/webapi/routing/v2/OntologiesRouteV2.scala

```diff
@@ -854,16 +844,16 @@ class OntologiesRouteV2(routeData: KnoraRouteData) extends KnoraRoute(routeData)
                   }
                   .getOrElse(List())
 
-              // validate the gui attributes by creating value objects
-              guiAttributes = maybeGuiAttributes.map(guiAttribute => GuiAttribute.make(guiAttribute)).toList
-
-              validatedGuiAttributes = Validation.validateAll(guiAttributes)
+              guiObject = GuiObject.makeFromStrings(maybeGuiAttributes, maybeGuiElement).fold(e => throw e.head, v => v)
 
-              // validate the combination of gui element and gui attribute by creating a GuiObject value object
-              guiObject: GuiObject = Validation
-                                       .validate(validatedGuiAttributes, validatedGuiElement)
-                                       .flatMap(values => GuiObject.make(values._1, values._2))
-                                       .fold(e => throw e.head, v => v)
+              requestMessage: CreatePropertyRequestV2 <- CreatePropertyRequestV2.fromJsonLD(
```

Comment:
I think there will be lots of merge conflicts here with my PR =/

Reply:
Yes, I thought that as well...